script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -89,28 +89,28 @@ Thinking about it, I've been gazing at this scenery for almost a year and a half
 This is the last time I will see this view on my train ride to school.
 I think back on the past seven years, tracing the landscape as I pass by.
 <Page25>
-When I was ten years old, I recovered from a serious accident that should have killed me instantly.
-I met Sensei.
-Then, I ended up living at the Arima household.
-And so the autumn of my second year of high school has arrived.
+When I was ten years old, I recovered from a serious accident that should have killed me on the spot, //The JP text uses commas instead of periods on this page, I think it flows better with the short sentences.
+Met Sensei,
+Ended up living at the Arima household,
+And finally, began the autumn of my second year of high school.
```

Comment:
If we're going to reflow to use commas, the start of these lines shouldn't be capitalized.

Reply:
I'm in agreement to change to commas